podcasts/Enumerations.swift

```diff
@@ -447,8 +445,6 @@ extension PlayerAction: AnalyticsDescribable {
     /// If false, the action will be hidden from the player shelf and overflow menu
     var isAvailable: Bool {
         switch self {
-        case .transcript:
-            return FeatureFlag.transcripts.enabled
         default:
```

Comment:
Should we just remove this method all together? Or keep if in the future we want to control available actions for the player?

Reply:
This predates Transcripts. It's been there since Bookmarks, so I think we can leave it there in case we need it.